climpred/bootstrap.py

```diff
@@ -364,14 +463,17 @@ def bootstrap_compute(
                 compute_persistence(smp_hind, reference, metric=metric, **metric_kwargs)
             )
     init = xr.concat(init, dim='bootstrap')
+    init = _ensure_loaded(init)
```

Comment:
I think all of these should be dropped. There's no reason we should force things into memory in our functions -- if the  user is using `dask` because the dataset is too large, we want to pass back lazy objects. Then they can run `.compute()` on their own time or might want to save them out as lazy objects or something.

Reply:
Fire twitch out the dask quantile Hack this wasn’t possible. Now it’s but I was afraid of too many calls to dask at the same time. That dask will figure this out. I will test this. 

My point in the current implementation is that all the bootstrapping iterations are embarrassingly parallel. However all of these need to be completed until we can continue the computation of the quantiles. Hope dask doesn’t get confused by that.

Reply:
Yes `dask` can handle it. It will set up a full graph of tasks to complete on each block (compute functions, shuffling, etc.) and then will execute them once the user runs `.compute()` on the returned lazy object.

Also I don't fully understand on the parallelism point. The way it is set up currently we are just encouraging chunking in spatial dimensions which triggers `dask` to chunk in space and *over each iteration* it is parallel. We don't have anything in place right now to parallelize the actual iterations, unless I am missing something. We'd need to use `multiprocessing` or some `dask` feature to actually parallelize over the iterations.